omod/src/main/resources/webModuleApplicationContext.xml

```diff
@@ -22,6 +22,10 @@
 
 	<!-- Add here beans related to the web context -->
 
-	<context:component-scan base-package="org.openmrs.module.appointments.web" />
+	<bean class="org.openmrs.ui.framework.StandardModuleUiConfiguration">

```

Comment:
Are we supporting the UI 2.x framework in this module?  Why is this being added?

Reply:
We actually should remove this.